libs/langgraph/langgraph/types.py

```diff
@@ -120,13 +123,18 @@ class RetryPolicy(NamedTuple):
 
 
 class CachePolicy(NamedTuple):
-    """Configuration for caching nodes."""
+    """
```

Comment:
```suggestion
    """Configuration for caching nodes.
```

Reply:
Could you restore the title? we shouldn't delete the doc-strings for types when adding versions